jupyterlab_latex/__init__.py

```diff
@@ -76,6 +76,8 @@ class LatexConfig(Configurable):
         help='The LaTeX command to use when compiling ".tex" files.')
     bib_command = Unicode('bibtex', config=True,
         help='The BibTeX command to use when compiling ".tex" files.')
+    synctex_command = Unicode('synctex', config=True,
```

Comment:
is this useful to configure? Is there any other command that one can use to run synctex?

Reply:
When I tested with `MiKTeX` I had to use `pdflatex.exe`, etc. I didn't want silly stuff like that being in the way of people using the endpoint.

That being said, I don't think there are any alternative commands than the `synctex` executable.

Reply:
to do synctex you had to run `pdflatex.exe`? that definitely makes sense, but if that's the case we should have a dynamic default… we're using traitlets afterall so why not.

Reply:
Maybe... it was kind of a pain for me to get `MikTeX` set up, and I don't really have much confidence that it is with the same defaults that others have.

Reply:
Sure, but I'm thinking it'd be good to know and the best way to find out is via bug reports. If we claim to have a default that works with MikTeX and it actually doesn't, people will respond and that'll allow us to have the people who actually want to use this figure it out with us (rather than us needing to do all that research on our own).

Reply:
If we go with dynamic defaults, can it be in a follow-up? I think it will require some significant testing...

Reply:
that sounds like a good idea!

I don't think we have any tests right now too… so we might want to consider that.

Reply:
Yeah, I think we are at the stage that tests would be worthwhile.

Reply:
Also, I was just trying to get `TeXLive` installed on Windows. It was like pulling teeth. I'll try again tomorrow.